### PR TITLE
Tfp/ben review

### PR DESCRIPTION
**Pull request by tfpauly**
Completes #88.

### CODE REVIEW

**Comment by tfpauly** [comment]
@chris-wood @theri could we get this reviewed before the interim?

File: draft-ietf-taps-transport-security.md [hunk 1]
```diff
@@ -579,6 +589,7 @@ between the record layer and the handshake layer.
 The record protocol can be signaled that it is being migrated to another transport or interface due to
 connection mobility, which may reset address and state validation and induce state changes such
 as use of a new Connection Identifier (CID).
+  - DTLS (version 1.3 only {{?I-D.ietf-tls-dtls13}})
```
Comment (renghardt) [inline draft-ietf-taps-transport-security.md:605]: I think in this case we should add an "x" to the summary table in Section 5.4. (We do the same for IKEv2.)

File: draft-ietf-taps-transport-security.md [hunk 2]
```diff
@@ -449,8 +458,9 @@ In the following sections, we provide abbreviations of the interface names to us
 Configuration interfaces are used to configure the security protocols before a
 handshake begins or the keys are negotiated.
 
-- Identities and Private Keys (IPK): The application can provide its identities (certificates) and private keys, or
-mechanisms to access these, to the security protocol to use during handshakes.
+- Identities and Private Keys (IPK): The application can provide its provide its identity,
```
Comment (renghardt) [inline draft-ietf-taps-transport-security.md]: nit: duplicate "provide its"

File: draft-ietf-taps-transport-security.md [hunk 3]
```diff
@@ -133,17 +133,20 @@ IETF protocols such as (D)TLS, as well as non-standard protocols such as gQUIC,
...
-omitted from this survey.
+unique design choices that have implications for and limitations on application usage. In contrast,
+protocols such as SSH {{?RFC4253}}, GRE {{?RFC2890}}, L2TP {{?RFC 5641}}, and ALTS {{ALTS}}
+are omitted since they do not provide interfaces deemed unique.
+
+Authentication-only protocols such as TCP-AO {{?RFC5925}} and IPsec Authentication Header (AH) {{?RFC4302}}
+are excluded from this survey. TCP-AO adds authentication to long-lived TCP connections, e.g., replay
+protection  with per-packet Message Authentication Codes. (TCP-AO obsoletes TCP MD5 "signature"
+options specified in {{?RFC2385}}.) One primary use case of TCP-AO is for protecting BGP connections.
+Similarly, AH adds per-datagram authenticationa and integrity, along with replay protection. Despite
```
Comment (renghardt) [inline draft-ietf-taps-transport-security.md]: nit: "authenticationa" (same as in the #99)

File: draft-ietf-taps-transport-security.md [hunk 4]
```diff
@@ -173,9 +176,12 @@ even where their interfaces to transport and applications are equivalent. Even b
 protocols have subtly different guarantees and vulnerabilities. Thus, any implementation needs to only
 use the set of protocols and algorithms that are requested by applications or by a system policy.
 
+Different security protocols also can use incompatible notions of peer identity and authentication, and
```
Comment (renghardt) [inline draft-ietf-taps-transport-security.md:183]: I would change the order and write "Different security protocols can also use", but I'm not sure either is more correct than the other.

**Review by renghardt — APPROVED** [review]
Looks good to me, modulo a few nits and a summary table update.

File: draft-ietf-taps-transport-security.md (hunk 3 above)
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]:
```suggestion
Similarly, AH adds per-datagram authentication and integrity, along with replay protection. Despite
```

File: draft-ietf-taps-transport-security.md (hunk 4 above)
Comment (chris-wood) [reply draft-ietf-taps-transport-security.md:183]: Eh, I think it's fine as is.

File: draft-ietf-taps-transport-security.md (hunk 2 above)
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]:
```suggestion
- Identities and Private Keys (IPK): The application can provide its identity,
```

File: draft-ietf-taps-transport-security.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-taps-transport-security.md:605]: Good catch!

**Review by chris-wood — APPROVED** [review]
Approved pending the table update fix from @theri!

**Comment by renghardt** [comment]
Updated the table and resolved the merge conflict.
I think this is good to go now.

**Pull request merged**